01-shapefile-attributes.Rmd

```diff
@@ -254,39 +239,44 @@ footpath_HARV<-lines_HARV[lines_HARV$TYPE == "footpath",]
 footpath_HARV
 #how many features in our new object
 length(footpath_HARV)
+```
+
+Our subsetting operation reduces the `features` count from 13 to 2. This means
+that only two lines have the attribute "TYPE=footpath".
 
+We can also plot our subsetted shapefiles.
+```{r plot-subset-shapefile}
 #plot just footpaths
 plot(footpath_HARV,
+     col=lines_HARV$OBJECTID, #color set by "OBJECTID" attribute
```

Comment:
Consider switching order of the subsections.  The plot with only 1 color appeared to plot just 1 footpath even thought there were two as the footpaths run end-to-end into each other.  Added color here to fix that misinterpretation, but then this comes before the Plot lines by Attribute Value where color is first introduced.